packages/admin/src/App.tsx

```diff
@@ -9,6 +10,7 @@ function App() {
       <Global styles={getGlobalStyle()} />
       <BrowserRouter>
         <Switch>
+          <Route path="/login" component={Login} />
           <Route path="/" component={Home} />
```

Comment:
```suggestion
          <Route path="/" exact component={Home} />
```
https://velopert.com/3417
exact를 붙여주는게 좋앙요~ 바꾸시는 김에 같이 바꿔주시면 감사하겠습니당